conjure-java-core/src/integrationInput/java/com/palantir/product/NameCollisionServiceEndpoints.java

```diff
@@ -63,23 +63,22 @@ private static final class IntEndpoint implements HttpHandler, Endpoint {
         public void handleRequest(HttpServerExchange exchange) throws IOException {
             AuthHeader authHeader = runtime.auth().header(exchange);
             String deserializer_ = deserializer.deserialize(exchange);
-            runtime.markers().param("com.palantir.logsafe.Safe", "deserializer", deserializer_, exchange);
             Map<String, String> pathParams =
                     exchange.getAttachment(PathTemplateMatch.ATTACHMENT_KEY).getParameters();
             String runtime_ = runtime.plainSerDe().deserializeString(pathParams.get("runtime"));
-            runtime.markers().param("com.palantir.logsafe.Safe", "runtime", runtime_, exchange);
+            runtime.markers().param("safe", "runtime", runtime_, exchange);
```

Comment:
We support both `com.palantir.logsafe.Safe` and `safe`, this PR standardizes on the short form.